### ISSUE TITLE
[otlLib] buildLigCaretList() with both coords and points is wrong

### ISSUE THREAD

**Issue by behdad** [behdad/fonttools#478]
The order of items in LigCaretList is significant, so this prototype cannot possibly be correct:
def buildLigCaretList(coords, points, glyphMap):

**Comment by brawer** [comment]
Agree, we should change it. Related bug in makeotf: https://github.com/adobe-type-tools/afdko/issues/102

For otlLib, what do you think would be a good API here? Perhaps just use the X coordinate of anchors? This would work for ligature carets in absolute coordinates, contour points, and device table entries. (The wording of Microsoft’s GDEF spec is rather unclear about X versus Y in ligature tables; in reality, it’s probably always X even though the GDEF spec claims otherwise?)
